CHANGES.md

```diff
@@ -159,6 +159,9 @@ Unreleased
 - Bump minimum version of the dune language for the melange syntax extension
   from 3.7 to 3.8 (#7665, @jchavarri)
 
+- Conservative implementation of lock file generation (#7732 partially fixes
```

Comment:
Since we're developing this feature in private, we shouldn't tell the user about this (yet)